static/sass/_pattern_search-panel.scss

```
@@ -3,8 +3,9 @@
   @include vf-u-text-muted;
 
   .p-search-panel {
-    margin-top: -1.2rem;
-    position : relative;
+    margin-top: 2.4rem;
```

I see a tiny gap at 2.4rem:
```suggestion
    margin-top: 2.3rem;
```

---

would this keep the text on the baseline grid? best to use the correct nudge variables in those calculations.